tests/contrib/redis/test_redis_cluster.py

```diff
@@ -141,14 +148,56 @@ def test_user_specified_service(self):
         span = spans[0]
         assert span.service != "mysvc"
 
-    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_REDIS_SERVICE="myrediscluster"))
-    def test_env_user_specified_rediscluster_service(self):
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v1"))
+    def test_user_specified_service_v1(self):
+        """
+        When a user specifies a service for the app
+            The rediscluster integration should use it.
+        """
+        # Ensure that the service name was configured
+        from ddtrace import config
+
+        assert config.service == "mysvc"
+
+        r = self._get_test_client()
+        Pin.get_from(r).clone(tracer=self.tracer).onto(r)
+        r.get("key")
+
+        spans = self.get_spans()
+        assert len(spans) == 1
+        span = spans[0]
+        assert span.service == "mysvc"
+
+    @TracerTestCase.run_in_subprocess(
+        env_overrides=dict(DD_REDIS_SERVICE="myrediscluster", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0")
+    )
+    def test_env_user_specified_rediscluster_service_v0(self):
```

Comment:
Nit: You could probably parametrize all of these if you wanted to avoid code reuse/clutter, e.g. https://github.com/DataDog/dd-trace-py/blob/1.x/tests/tracer/test_propagation.py#L451-L476

But up to you!

Reply:
I looked into this and didn't find an easy way to parametrize a decorator =/